uproot_methods/classes/TLorentzVector.py

```diff
@@ -203,7 +203,7 @@ def mt(self):
 
     @property
     def rapidity(self):
-        return self.awkward.numpy.log((self.t + self.z) / (self.t - self.z)) / 2.0
```

Comment:
Right, multiplication is faster than division. But bit-shifting is even faster...  :)

Reply:
Well, masking out the mantissa and bit-shifting that may be slower.

Reply:
Ah, forgot that this was floating point...